scivision_test_plugin/model.py

```diff
@@ -15,13 +22,29 @@ def __init__(self):
         pass
 
     def do_something(
-        self,
-        preserve_range: bool = True,
-        sigma: float = 10.0,
-        image: np.ndarray = np.empty((64, 64, 1)),
+            self,
+            preserve_range: bool = True,
+            sigma: float = 10.0,
+            image: np.ndarray = np.empty((64, 64, 1)),
     ) -> np.ndarray:
         return gaussian(image, sigma=sigma, preserve_range=preserve_range)
 
 
+class ImageNetModel:
+    def __init__(self):
+        self.pretrainedModel, self.preprocess_input = Classifiers.get('resnet18')
```

Comment:
```suggestion
    def __init__(self, model_name:str = "resnet18"):
        model, self.preprocess_input = Classifiers.get(model_name)
        
        # can build the model here, so we can reuse the prediction function
        self.pretrained_model = model(
            input_shape=(224, 224, 3), weights="imagenet", classes=1000
        )
```

Reply:
Also, `input_shape`, `weights` and `classes` could become arguments here. We could specify those in the config file in the same way as `model_name`

Reply:
ok, looks ok for me.